PyInstaller/depend/bindepend.py

```diff
@@ -875,6 +876,36 @@ def get_python_library_path():
     # Applies only to non Windows platforms.
 
     if is_unix:
+
+        # Search if it is an Anaconda distribution
+        if re.findall('/anaconda', sys.executable):
+
+            # Relative anaconda path to the lib
+            rel_lib_path = 'lib/libpython3.7m.so.1.0'
+
+            # Split the path of the python executable
+            # to determine what the full anaconda name is
+            # and what paths preceed it.
+            splits = sys.executable.split('/anaconda')
+
+            # The preceding path to the anaconda directory
+            pre_path = splits[0]
+
+            # Split the remainder to seperate any other paths
+            back_splits = splits[1].split('/')
```

Comment:
Looks like you are basically calculating `sys.exec_prefix`? (Beside of again not working in e.g. Windows.)

Please give an example of the full paths you have and want to receive.

Reply:
I have used compat.base_prefix instead. Thanks